src/usecases/user/payments.js

```diff
@@ -0,0 +1,15 @@
+const User = require("../../models/user").model;
+const Pickup = require("../../models/pickup").model;
+
+const getAllPayments = (data) => {
+
```

Comment:
Don't leave empty use case

Reply:
This whole section of the project is on pause or obsolete as it will be replaced by stripe api implementation